components/match2/commons/match_group_input.lua

```diff
@@ -119,13 +119,27 @@ function MatchGroupInput.readBracket(bracketId, args, options)
 				.. MatchGroupUtil.matchIdFromKey(match.loserto)
 		end
 
-		-- Kick bracketData.thirdplace if no 3rd place match
-		if bracketData.thirdplace ~= '' and not args.RxMTP then
-			bracketData.thirdplace = ''
+		-- Remove bracketData.thirdplace if no 3rd place match
+		if not args.RxMTP then
+			bracketData.thirdplace = nil
 		end
-		-- Kick bracketData.bracketreset if no reset match
-		if bracketData.bracketreset ~= '' and not args.RxMBR then
-			bracketData.bracketreset = ''
+		bracketData.thirdplace = bracketData.thirdplace or ''
```

Comment:
Could be simplified (if we want) to 
bracketData.thirdplace = args.RxMTP and bracketData.thirdplace or ''

Reply:
It's not the same - I wan't to store `''` when bracketData.thirdplace is not present. This is temporary, eventually it will be changed so it will store nothing instead of `''`.

Reply:
Add a TODO

Reply:
ok